scipy/linalg/_solvers.py

```diff
@@ -150,7 +150,7 @@ def _solve_discrete_lyapunov_bilinear(a, q):
 
 def solve_discrete_lyapunov(a, q, method=None):
     """
-    Solves the discrete Lyapunov equation :math:`A'XA-X=-Q`.
+    Solves the discrete Lyapunov equation :math:`AXA^H - X = -Q`.
```

Comment:
If Q is indeed required to be Hermitian, then the word "square" should be dropped.